app/src/routes/login/login.vue

```diff
@@ -78,7 +79,11 @@ export default defineComponent({
 			providerOptions.value = providers.value
 				.filter((provider) => !AUTH_SSO_DRIVERS.includes(provider.driver))
 				.map((provider) => ({ text: formatTitle(provider.name), value: provider.name }));
-			providerOptions.value.unshift({ text: t('default_provider'), value: DEFAULT_AUTH_PROVIDER });
+			if (!disableDefault.value) {
+				providerOptions.value.unshift({ text: t('default_provider'), value: DEFAULT_AUTH_PROVIDER });
+			} else {
+				providerSelect.value = providerOptions.value[0]?.value;
```

Comment:
I think this is supposed to be 

```suggestion
				provider.value = providerOptions.value[0]?.value;
```

h/t to @aidenfoxx for pointing that out

Reply:
This does not seem to work, as it does not update the driver and therefore the wrong form is rendered.

Reply:
@rijkvanzanten Is it fine like this now? I can change this, but it would produce redundant code...

Reply:
I think it's fine. It was mostly that I didn't understand how `getters/setters` so I raised the question. But it seems correct that you can do `providerSelect.value = thing` and it'll trigger the setter function, looking at the docs.

So I think you can resolve this.